### DISCUSSION TITLE
Will the image will be pulled every minute?

### DISCUSSION THREAD

**Discussion by Deshdeepak1** [containrrr/watchtower#1023] [Q&A] [answered]
Will the image will be puled every minute and then checked if it has changed or not, or will only be pulled if it has changed ?

**Comment by piksel** [comment, marked as answer]
If you don't specify any interval, the default is to check every 24h. 

If the repository supports it, it will first ask for a hash of the latest image from the repo and compare it with the one used locally. If those does not match, or if the repo does not support such requests, it will do a pull.
